coalib/bears/requirements/PythonRequirement.py

```diff
@@ -77,12 +91,14 @@ def multiple(cls, *args):
         for requirement in args:
             if isinstance(requirement, str):
                 reqs.append(cls(requirement),)
-            elif isinstance(requirement, tuple):
+            elif isinstance(requirement, tuple) or isinstance(requirement,
+                                                              list):
                 try:
                     name, version = requirement
                     reqs.append(cls(name, version),)
```

Comment:
Probably not related to this PR ... but why is there `reqs.append(cls(name, version),)` ? Why is there a comma ?